### PR TITLE
Summarize the number of using gems

### PR DESCRIPTION
**Pull request by hbd225**
For this issue bundler/bundler-features#33
Thanks for making great gem :)

### CODE REVIEW

File: lib/bundler/cli/install.rb [hunk 1]
```diff
@@ -105,6 +105,7 @@ def run
       Installer.install(Bundler.root, definition, options)
       Bundler.load.cache if Bundler.app_cache.exist? && options["cache"] && !Bundler.settings[:frozen]
 
+      Bundler.ui.confirm "Using #{Installer.using_gems.size} already installed gems" if Installer.using_gems?
```
Comment (segiddins) [inline lib/bundler/cli/install.rb]: just use `Installer.using_gems.size > 0`?

Comment (hbd225) [inline lib/bundler/cli/install.rb]: Fixed.

**Comment by segiddins** [comment]
:+1:

**Comment by indirect** [comment]
@homu r+

**Pull request merged**